extensions/blocks/pinterest/pinterest.php

```diff
@@ -180,10 +218,28 @@ function render_amp_pin( $attr ) {
  * @return string
  */
 function load_assets( $attr, $content ) {
+	if ( ! jetpack_is_frontend() ) {
+		return $content;
+	}
 	if ( Blocks::is_amp_request() ) {
 		return render_amp_pin( $attr );
 	} else {
+		$url  = $attr['url'];
+		$type = pin_type( $url );
+
+		if ( ! $type ) {
+			return '';
+		}
+
 		wp_enqueue_script( 'pinterest-pinit', 'https://assets.pinterest.com/js/pinit.js', array(), JETPACK__VERSION, true );
-		return $content;
+		return sprintf(
+			'
+			<div class="wp-block-jetpack-pinterest">
```

Comment:
Oh, I just remembered we can use `Blocks::classes` here, as we did in the Donations block. That way, any other additional class added by users through the sidebar is automatically added:

<img width="281" alt="Screen Shot 2020-10-30 at 12 38 53" src="https://user-images.githubusercontent.com/1233880/97700926-e397b700-1aac-11eb-9db4-ed2cee2ef5b8.png">

Reply:
Done in https://github.com/Automattic/jetpack/pull/17601/commits/26c28ae4d7066bd958d218ca56a359137e84886c